dask/array/reductions.py

```diff
@@ -333,8 +334,33 @@ def moment_agg(data, order=2, ddof=0, dtype='f8', sum=np.sum, **kwargs):
 
 def moment(a, order, axis=None, dtype=None, keepdims=False, ddof=0,
            split_every=None):
-    if not isinstance(order, int) or order < 2:
-        raise ValueError("Order must be an integer >= 2")
+
+    if not isinstance(order, int) or order < 0:
+        raise ValueError("Order must be an integer >= 0")
+
+    if order == 0:
+        # When order equals 0, the result is 1, by definition.
+        shape = list(a.shape)
+        del shape[axis]
```

Comment:
Previously this worked with tuple axis. Perhaps:

```python
if order < 2:
    reduced = a.sum(axis=axis)   # get reduced shape and chunks
    if order == 0:
        return ones(reduced.shape, chunks=reduced.chunks, dtype='f8')
    return zeros(reduced.shape, chunks=reduced.chunks, dtype='f8')
```

This also returns a scalar for reductions that result in a scalar (fixing your `TODOs`).